webpack/assets/javascripts/react_app/common/AwaitedMount.js

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+import { useState, useEffect } from 'react';
+import store from '../redux';
+import componentRegistry from '../components/componentRegistry';
+
+// Mounts a component after all plugins have been imported to make sure that all plugins are available to the component
+export const AwaitedMount = ({ component, data, flattenData }) => {
```

Comment:
there's a warning on every page about `Component name already taken:` 
from the stack trace it's originated from this file.

![image](https://github.com/theforeman/foreman/assets/26363699/b1ae6e42-dde2-4166-a1e0-3ea5b24771b7)

![image](https://github.com/theforeman/foreman/assets/26363699/5900ce80-7bc4-44d0-9843-c0ade0f5ac20)

Reply:
Was this compiled with foreman-js 12.2.2? If so, that's the real culprit, not this line.
12.2.3 should fix everything.

Reply:
should we drop the version pin to v12.2.0 now? https://github.com/theforeman/foreman/pull/9834/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R24 ?

Reply:
12.2.3 should be fine. only .1 and .2 are broken.

I'll drop the pin, good catch

Reply:
yeah looks like this is fixed in the latest commit here and using v12.2.3

Reply:
apologies, there was an issue that plugins weren't compiling well using `bundle exec rake webpack:compile`
it worked with `npx webpack  --config config/webpack.config.js`

and now the issue is visible again